component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/util/TOTPUtil.java

```diff
@@ -309,12 +309,40 @@ private static String getEncodingMethodFromRegistry(String tenantDomain, Authent
 		return encodingMethod;
 	}
 
+	/**
+	 * Get stored time step size.
+	 *
+	 * @param tenantDomain Tenant domain name.
+	 * @return Time step size.
+	 * @throws AuthenticationFailedException On Error while getting value for time step size from registry.
+	 */
+	public static Long getTimeStepSize(String tenantDomain) throws AuthenticationFailedException {
+
+		Long timeStepSize;
+		if (tenantDomain.equals(TOTPAuthenticatorConstants.SUPER_TENANT_DOMAIN)) {
```

Comment:
if(TOTPAuthenticatorConstants.SUPER_TENANT_DOMAIN.equals(tenantDomain)) {} to avoid possible null pointers

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/77/commits/361df60ef387822f489ddafe9bccd517f32bf053.